### ISSUE TITLE
[SITE] Nav Bar is missing on the mobile view of the page

### ISSUE THREAD

**Issue by asubedy** [layer5io/getnighthawk#258]
#### Current Behavior
When we open the site in mobile view, the navigation menu (hamburger menu), disappears.

#### Expected Behavior
There should be an navigation menu, that allows us to open the menu and switch to different section of the site

---

#### Contributor Resources

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
  - 🎨 Wireframes and designs for Nighthawk Site in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by sayan011** [comment]
Hey @Chadha93 ,can I work on this issue?

**Comment by siddhantprateek** [comment]
@Chadha93, I would like to try this issue.

**Comment by siddhantprateek** [comment]
I can see the changes locally, but it's not visible in the build.

**Comment by subhradwip01** [comment]
Is this issue solved? If not I would like to work on that issue

**Comment by siddhantprateek** [comment]
Yes, go ahead, but I think changes are already made. It's not visible in the build process.

**Comment by kunalchhabra37** [comment]
I would Like to Work on this Issue.

**Comment by subhradwip01** [comment]
Go ahead...problem in build...otherwise it is ok

**Comment by harshitphoenix** [comment]
Is this issue still there? If yes and no one is on it, I would like to take it.
